tests/integration/cartography/intel/aws/test_dynamodb.py

```diff
@@ -0,0 +1,18 @@
+import cartography.intel.aws.dynamodb
+import tests.data.aws.dynamodb
+
+TEST_ACCOUNT_ID = '000000000000'
+TEST_REGION = 'us-east-1'
+TEST_UPDATE_TAG = 123456789
+
+
+def test_load_dynamodb(neo4j_session):
```

Comment:
Non-blocking: For extra-extra confidence that your nodes are getting loaded with the members that you expect, you _could_ add an integration test like this: https://github.com/lyft/cartography/pull/76/files#diff-a58e210adeb9fd885c695de8a51f486eR32

I'll let @ecdavis decide if this is necessary; I'm good either way, but it could be a good thing to have.

Reply:
Wrote some basic tests in the format. Should be good!